tests/test_config.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+
+from mock import MagicMock, Mock, patch
+from patroni.config import Config
+from six.moves import builtins
+
+
+class TestConfig(unittest.TestCase):
+
+    @patch('os.path.isfile', Mock(return_value=True))
+    @patch('json.load', Mock(side_effect=Exception))
+    @patch.object(builtins, 'open', MagicMock())
+    def setUp(self):
+        self.config = Config(config_env='postgresql: {data_dir: foo}')
+
+    def test_reload_local_configuration(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/cyberdemn/patroni/file/2630856527/issues/source?bid=3127594&fileBranchId=3318079#l16)